jujugui/static/gui/src/app/components/boolean-config/test-boolean-config.js

```diff
@@ -18,7 +18,7 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 'use strict';
 
-const juju = {components: {}}; // eslint-disable-line no-unused-vars
+var juju = {components: {}}; // eslint-disable-line no-unused-vars
```

Comment:
Haha, so it's because the way the test runner works. It loads every file into a single instance so it was trying to redefine juju every time. Const and let don't allow that but var does. This way we can still load the files individually or all at once as usual.

This didn't cause an issue until now because const was being transpiled to var anyways.

An alternative fix was to make it a conditional. But to do that I'd have to modify every test file instead of just updating a few.